docs/userguide/networking/get-started-overlay.md

```diff
@@ -91,7 +93,7 @@ that machine options that are needed by the `overlay` network driver.
 
 		$ docker-machine create \
 		-d virtualbox \
-		--swarm --swarm-image="swarm" --swarm-master \
+		--swarm --swarm-master \
```

Comment:
Actually, I did that specifically so they could see the relationship between the `--swarm` flag and the next. I don't think it is difficult to read and it certainly is a PIA to copy if they are all on their own line.